gradle/wrapper/gradle-wrapper.properties

```diff
@@ -5,4 +5,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.2-bin.zip
```

Comment:
With the change here do we have to make the same change across all the repos? I don’t know how many are on the old version. I’m assuming they will suffer the same ssl error, no?

Reply:
I'd be in favor of it but have not too much insight of what might break, so I rely on your advise...

Reply:
https://github.com/apache/incubator-openwhisk/pull/4111/files#diff-fd9d61e37d939c941e7a62077f38d62dR6 the openwhisk repo is already on this version.